tests/strategies/settlement/test_settlement_strategy.py

```diff
@@ -0,0 +1,120 @@
+import uuid
+from unittest.mock import Mock, MagicMock
+
+import pytest
+from d3a_interface.constants_limits import ConstSettings
+from pendulum import datetime, duration
+
+from d3a.models.market.market_structures import Bid, Offer, Trade
+from d3a.models.market.two_sided import TwoSidedMarket
+from d3a.models.strategy.load_hours import LoadHoursStrategy
+from d3a.models.strategy.pv import PVStrategy
+from d3a.models.strategy.settlement_strategies import SettlementMarketStrategy
```

Comment:
```suggestion
import pytest
from d3a.models.market.market_structures import Bid, Offer, Trade
from d3a.models.market.two_sided import TwoSidedMarket
from d3a.models.strategy.load_hours import LoadHoursStrategy
from d3a.models.strategy.pv import PVStrategy
from d3a.models.strategy.settlement_strategies import SettlementMarketStrategy
from d3a_interface.constants_limits import ConstSettings
from pendulum import datetime, duration
```

NIT: A recommendation of an `optimise imports`

Reply:
This is contrary to PEP8 though, and we have decided to follow this (pendulum and d3a-interface should precede d3a. Also lines should separate the 3 blocks of imports. Actually, the original commit is fine as is)

Reply:
https://user-images.githubusercontent.com/33482427/131500822-0705a05f-8eb2-4228-bf97-979107219df7.mov

Okay, may be PyCharm's default optimise import needs to be customised then.

Reply:
Hmm, that's very strange, and it does not happen for me. I think the problem is not Pycharm per se, but that for some reason it recognises the d3a package as a third-party package, while in reality is the main package of the project. Can it be that you have multiple repos (e.g. d3a, d3a-web, d3a-interface) on the same Pycharm project ? Because in this case I could foresee such a behavior. Let's continue to discuss this on Slack.

Reply:
> Can it be that you have multiple repos (e.g. d3a, d3a-web, d3a-interface) on the same Pycharm project ?

Not really, they are in different PyCharm projects. Anyways, marking this comment as resolved.